server/candlepin.spec.tmpl

```diff
@@ -87,6 +88,11 @@ SELinux policy module supporting candlepin
 
 %prep
 %setup -q %{SOURCE0}
+%if 0%{?rhel} == 8
+pathfix.py -pni "%{__python3} %{py3_shbang_opts}" . code/setup/cpdb code/setup/cpsetup
```

Comment:
Please remove '.' from the arguments. Otherwise, this ends up evaluating all the source files and looking for shebangs (harmless, but not necessary).